manifests/d/DuckStudio/ChineseGit/2.5/DuckStudio.ChineseGit.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DuckStudio.ChineseGit
+PackageVersion: "2.5"
+InstallerLocale: zh-CN
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /verysilent /suppressmsgboxes
+  SilentWithProgress: /verysilent
+UpgradeBehavior: install
+Commands:
+- cngit
+Protocols:
+- http
+- https
+ReleaseDate: 2024-06-01
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/DuckDuckStudio/Chinese_git/releases/download/v2.5/Chinese_git_Setup_v2.5.exe
+  InstallerSha256: CF79850D8436B0A69641D5413FA5D5D41B7AAE0BACB125022D315281445874FB
```

```suggestion
  InstallerSha256: CF79850D8436B0A69641D5413FA5D5D41B7AAE0BACB125022D315281445874FB
  AppsAndFeaturesEntries:
  - DisplayVersion: 'v2.5'
```
This is necessary since the PackageVersion doesn't match Control Panel